### PR TITLE
CI pipeline template for building sample solutions

### PR DESCRIPTION
**Pull request by kythant**
CI pipeline template for building sample solutions

Test Run:
https://dev.azure.com/ms/ProjectReunion/_build/results?buildId=260854&view=logs&j=232dc563-2327-5715-c55f-22c3ab542119

I've also moved the non-building samples into the quarantine folder. Once fixed, they should be moved back into the Samples folder.

### CODE REVIEW

**Comment by erik0686** [comment]
NICEEEEE

File: WindowsAppSDK-SamplesCI.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            buildConfiguration: 'Release'
+          Release_x64:
+            buildPlatform: 'x64'
+            buildConfiguration: 'Release'
+          Debug_x86:
+            buildPlatform: 'x86'
+            buildConfiguration: 'Debug'
+          Debug_x64:
+            buildPlatform: 'x64'
+            buildConfiguration: 'Debug'
```
Comment (angelazhangmsft) [inline WindowsAppSDK-SamplesCI.yml:21]: Are we able to add arm64 here?

**Review by angelazhangmsft — COMMENTED** [review]
This is great!! Another question I had is if this pipeline will allow us to build the samples on internal versions of WinAppSDK?

**Review by BenJKuhn — APPROVED** [review]
Agree with Angela's comment, if we can build Arm64, let's do that as well. Otherwise looks good.

**Review by erik0686 — APPROVED** [review]

File: WindowsAppSDK-SamplesCI.yml (hunk 1 above)
Comment (kythant) [reply WindowsAppSDK-SamplesCI.yml:21]: I can try! Another PR incoming.

**Comment by kythant** [comment]
> This is great!! Another question I had is if this pipeline will allow us to build the samples on internal versions of WinAppSDK?

To answer your question. Not yet. But I am hoping the internal version of this pipeline will soon.

**Pull request merged**